rfcs/testdriver_in_other_types.md

```diff
@@ -0,0 +1,150 @@
+| Various [`css/`][5] | `Actions().pointerMove()` | Exercise `:hover` styles |
+| TBD | TBD | Editing caret rendering |
+| TBD | TBD | Spelling/grammar/etc markers |
+
+* The `customizable-select/` tests are confined to [Chromium's
+  `wpt_internal/`][6] because only Chromium's `run_web_tests.py` and
+  [`testdriver-vendor.js`][7] can run them.
+  These tests are effectively not portable to other vendors.
+* The other tests have already landed in web-platform-tests/wpt, but
+  [time out in all browsers][1] when run with [wptrunner], WPT's canonical test
+  harness.
+  These tests don't provide reliable coverage.
+
+[have expressed interest]: https://github.com/web-platform-tests/wpt/issues/13183
+[wptrunner]: https://web-platform-tests.org/tools/wptrunner/README.html
+[testharness tests]: https://web-platform-tests.org/writing-tests/testharness.html
+[reftests]: https://web-platform-tests.org/writing-tests/reftests.html
+[0]: https://web-platform-tests.org/writing-tests/testdriver.html
+[1]: https://wpt.fyi/results/fullscreen/rendering?run_id=6207808991395840&run_id=5099292293595136&run_id=5124412349349888&run_id=5184801065926656
+[2]: https://developer.mozilla.org/en-US/docs/Web/API/Element/requestFullscreen#security
+[3]: https://chromium.googlesource.com/chromium/src/+/main/third_party/blink/web_tests/wpt_internal/html/semantics/forms/the-select-element/customizable-select/
+[4]: https://developer.mozilla.org/en-US/docs/Web/API/HTMLSelectElement/showPicker#security_considerations
+[5]: https://github.com/search?q=repo%3Aweb-platform-tests%2Fwpt+path%3Acss+testdriver.js+%2Flink.*rel%3D.*match%2F&type=code
+[6]: https://chromium.googlesource.com/chromium/src/+/main/third_party/blink/web_tests/wpt_internal/README.md
+[7]: https://chromium.googlesource.com/chromium/src/+/main/third_party/blink/web_tests/resources/testdriver-vendor.js
+
+## Details
+
+### Usage
+
+Using testdriver.js in non-testharness tests will be almost identical to the
+testharness case.
+Simply add `<script src=/resources/testdriver(-vendor).js>` before other
+scripts:
+
+```html
+<html class=reftest-wait>
+<link rel=match href=/common/blank.html>
+<script src=/resources/testdriver.js></script>
+<script src=/resources/testdriver-vendor.js></script>
+<script>
+  test_driver.bless("complete test",
+    () => document.documentElement.classList.remove("reftest-wait"));
+</script>
+```
+
+There are no other changes to (print-)reftest or crashtest markup.
+
+Because the [default completion behavior][completion-behavior] is not
+well-ordered with respect to arbitrary testdriver calls, testdriver in
+non-testharness tests will only be supported with `class=(ref)test-wait`.
+
+[completion-behavior]: https://web-platform-tests.org/writing-tests/reftests.html#controlling-when-comparison-occurs
+
+### Manifest Format Changes
+
+Similar to testharness manifest items, items for the other types may set a
```

Comment:
This shouldn't be "may", but "will" or "must".

Reply:
Reworded a bit